packages/perseus/src/components/sortable.tsx

```diff
@@ -757,7 +727,10 @@ class Sortable extends React.Component<SortableProps, SortableState> {
             throw new Error(`index ${index} out of bounds`);
         }
 
-        const nextItems = _.clone(items);
+        // NOTE(kevinb): we cast to `Array<SortableItem>` to remove
+        // the readonly-ness of the cloned value.  `clone()` should
+        // probably not being return a readonly value.
+        const nextItems = _.clone(items) as Array<SortableItem>;
```

Comment:
This was a bit annoying.  I'd rather not have to cast the return type of `clone()`.

Reply:
That makes sense.  Good call!